core/lane.py

```diff
@@ -3211,12 +3213,6 @@ def explain(self):
         return lines
 
 
-Library.lanes = relationship(
-    "Lane",
-    backref="library",
-    foreign_keys=Lane.library_id,
-    cascade="all, delete-orphan",
-)
 DataSource.list_lanes = relationship(
```

Comment:
Since we removed the above bit of nightmare surprise database modelling, could we remove this nightmare surprise database modelling as well?

Reply:
Removed 👍